src/sdk/src/fragment-components/a-combobox.tsx

```diff
@@ -273,7 +273,7 @@ export const componentInfo: ComponentInfo = {
 	component: AComboBox,
 	settingsUI: AComboBoxSettingsUI,
 	codeUI: AComboBoxCodeUI,
-	keywords: ['ComboBox', 'filterable', 'multiselect'],
+	keywords: ['combobox'],
```

Comment:
filterable and multiselect should probably stay because people might search for those when looking for combobox